packages/amplify-util-uibuilder/src/__tests__/generateComponents.test.ts

```diff
@@ -63,29 +77,29 @@ describe('can generate components', () => {
           },
         }),
       }),
-    });
-    utilsMock.generateUiBuilderComponents = jest.fn().mockReturnValue(schemas.entities);
-    utilsMock.generateUiBuilderThemes = jest.fn().mockReturnValue(schemas.entities);
-    utilsMock.generateUiBuilderForms = jest.fn().mockReturnValue(schemas.entities);
-    utilsMock.getAmplifyDataSchema = jest.fn().mockReturnValue(undefined);
-    utilsMock.generateAmplifyUiBuilderIndexFile = jest.fn().mockReturnValue(true);
-    utilsMock.generateAmplifyUiBuilderUtilFile = jest.fn().mockReturnValue(true);
+    }));
+    utilsMock.generateUiBuilderComponents = jest.fn().mockImplementation(() => schemas.entities);
+    utilsMock.generateUiBuilderThemes = jest.fn().mockImplementation(() => schemas.entities);
+    utilsMock.generateUiBuilderForms = jest.fn().mockImplementation(() => schemas.entities);
+    utilsMock.getAmplifyDataSchema = jest.fn().mockImplementation(() => undefined);
+    utilsMock.generateAmplifyUiBuilderIndexFile = jest.fn().mockImplementation(() => true);
+    utilsMock.generateAmplifyUiBuilderUtilFile = jest.fn().mockImplementation(() => true);
```

Comment:
`jest.mockReturnValue` should be the same as passing an arrow function to `jest.mockImplementation` as far as I know